src/pages/adminDistribution/tests-page-admin.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import Layout from '../../components/Layout/Layout.js';
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
```

Comment:
use such a style of imports

import { AppBar, Toolbar, bla bla bla, IconButton, Avatar, Badge, blablabla, Drawer } from '@material-ui/core';